{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "85ac2207_23e7aed3",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1945317
      },
      "writtenOn": "2023-04-14T16:08:26Z",
      "side": 1,
      "message": "Detailed explanation is in comment 39 of this bug.",
      "revId": "eb54a6022d559b51d364200ef3d5fc4831cc9f85",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}